frontend/src/common.js

```diff
@@ -3,8 +3,28 @@ window.Raven.config(window.dsn).install();
 window.dsn = undefined;
 
 const testPilotGA = require('testpilot-ga');
+const {gcmCompliant, sendEvent} = require('./utils');
 window.analytics = new testPilotGA({
   an: 'Firefox Send',
   ds: 'web',
   tid: window.trackerId
 });
+
+gcmCompliant().catch(err => {
+  $('#page-one').attr('hidden', true);
+  $('#download').attr('hidden', true);
+  sendEvent('sender', 'unsupported', {
```

Comment:
Below this was "recipient" in /frontend/src/download.js (but "sender" in the upload.js).
Is this fine here as "sender" always?